src/i18n/translations/en.json

```diff
@@ -0,0 +1,3 @@
+{
+  "helloWorld": "Hello World"
```

Comment:
I recommend you to create an object in this way: 
`{
 helloWold: "Hello World"
}`

Reply:
I changed JSON files to TS files. Is it what you meant?